geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/collections/SizableOrderStatisticsTree.java

```diff
@@ -731,6 +741,25 @@ private int count(Node<E> node) {
     return leftTreeSize + 1 + rightTreeSize;
   }
 
+  void updateSizeInBytes(E element, boolean isAdd) {
+    int sizeChange;
+    if (element instanceof Sizeable) {
```

Comment:
I don't think we do this anywhere else, but I did it here to avoid having to use ReflectionSingleObjectSizer for objects that are already keeping track of their size, since that would reduce overhead (I think). Your suggestion is a good one though, so I'll implement it here.